<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-CGUJBJB4" failures="1" tests="3" name="TestngPractice" time="0.095" errors="0" timestamp="2024-02-14T10:39:59 IST">
  <testcase classname="Testngpract.ListenersDemo" name="skipfail" time="0.006">
    <skipped/>
  </testcase> <!-- skipfail -->
  <testcase classname="Testngpract.ListenersDemo" name="testfail" time="0.021">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Testngpract.ListenersDemo.testfail(ListenersDemo.java:45)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- testfail -->
  <testcase classname="Testngpract.ListenersDemo" name="testpass" time="0.0"/>
</testsuite> <!-- TestngPractice -->
